### PR TITLE
Setup Bookstore App

### PR DESCRIPTION
**Pull request by zunairkhan811**
#SETTING UP BOOK ENVIRONMENT
IMPLEMENTATION
-Contains individual book state, as well as a button to delete the book
-Contains a form to create new books
-Contains navigation to link

### CODE REVIEW

File: src/components/Todo.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+const TodoList = () => {
```
Comment (VICTOR-CHUKWU) [inline src/components/Todo.js]: - [x] I think this is a book store app so your naming should align, the name of this component should be book related eg  `Booklist.js`

File: src/components/Todo.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      <div className="container">
+        {data.map((item) => (
+          <div key={item.id}>
+            <h2>{item.title}</h2>
+            <p>
+              By
+              {item.author}
+            </p>
+            <button type="button" id="remove">Remove</button>
+          </div>
```
Comment (VICTOR-CHUKWU) [inline src/components/Todo.js]: - [x] according to project requirement, kindly make this a component that takes props and import here

File: src/components/Todo.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        ))}
+        <input type="text" id="title" placeholder="Your Title Here..." />
+        <br />
+        <input type="text" id="name" placeholder="Your Name Here..." />
+        <br />
+        <button type="submit">Add Book</button>
```
Comment (VICTOR-CHUKWU) [inline src/components/Todo.js]: - [x] also kindly make this a component and import it here

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @zunairkhan811  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TadesseDev — APPROVED** [review]
### Hi @zunairkhan811 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**